include/acl_selectors.php

```diff
@@ -388,13 +388,9 @@ function populate_acl($user = null, $show_jotnets = false) {
 }
 
 function construct_acl_data(App $a, $user) {
-
-	$arr = array('user' => $user);
-	call_hooks('construct_acl_data', $arr);
-
-	if (isset($arr['cancel'])) {
-		return;
-	}
+	// This function is now deactivated. It seems as if the generated data isn't used anywhere.
+	// We will remove the function completely before the release of 3.5.3 when there won't be any issues.
+	return;
```

Comment:
I want to keep it for the time until the release that way. This would make the reenabling more easy. When at the time for the release there had been no issues at all then we should remove it completely.

Because of this I wouldn't like to make more work with it.